golang/server/server.go

```diff
@@ -137,16 +164,21 @@ func (server MinimalGRPCServer) RunUntilStopped(stopper <-chan struct{}) error {
 	case <-time.After(server.stopGracePeriod):
 		logrus.Warnf("gRPC server failed to stop gracefully after %v; hard-stopping now...", server.stopGracePeriod)
```

Comment:
the mux server could be the one taking time now. maybe we can make this more generic. same for line 168

Reply:
i don;t think so change of error message is needed; because at the end the core responsibility is that it's grpc server; and has proxy on top of it.